forte/base_classes/active_space_method.h

```diff
@@ -39,6 +40,10 @@
 #include "psi4/libmints/matrix.h"
 #include "psi4/libmints/vector.h"
 
+namespace ambit {
+class BlockedTensor;
+}
+
```

Comment:
Should we also put `BlockedTensor` to include?

Reply:
done